tests/Digdir.Domain.Dialogporten.WebApi.Unit.Tests/Features/V1/SwaggerSnapshotTests.cs

```diff
@@ -1,53 +1,54 @@
+#if DEBUG
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously.
+#endif
+
 using System.Diagnostics.CodeAnalysis;
 using System.Text.Json;
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.AspNetCore.Mvc.Testing;
 
-namespace Digdir.Domain.Dialogporten.WebApi.Integration.Tests.Features.V1;
+namespace Digdir.Domain.Dialogporten.WebApi.Unit.Tests.Features.V1;
 
-public class SwaggerSnapshotTests : IClassFixture<WebApplicationFactory<Program>>
+public class SwaggerSnapshotTests
 {
-    private readonly WebApplicationFactory<Program> _webApplicationFactory;
-
-    public SwaggerSnapshotTests(WebApplicationFactory<Program> webApplicationFactory)
-    {
-        _webApplicationFactory = webApplicationFactory;
-    }
 
     [Fact]
     public async Task FailIfSwaggerSnapshotDoesNotMatch()
     {
+#if RELEASE
         // Arrange
         // This test checks for changes against the published version of the swagger.verified.json file
         // The file is located at /docs/schema/ on the solution root
         // Commiting a change to this file will trigger a build and publish
         // of the npm package located in the same folder
         var rootPath = Utils.GetSolutionRootFolder();
         var swaggerPath = Path.Combine(rootPath!, "docs/schema/V1");
-
-        var client = _webApplicationFactory
-            .WithWebHostBuilder(builder => builder.UseEnvironment("test"))
-            .CreateClient();
-
+        var newSwaggerPath = Path.Combine(rootPath!, "src/Digdir.Domain.Dialogporten.WebApi/bin/Release/net9.0/swagger.json");
```

Comment:
```suggestion
#if NET9_0
        var newSwaggerPath = Path.Combine(rootPath!, "src/Digdir.Domain.Dialogporten.WebApi/bin/Release/net9.0/swagger.json");
#endif
```

Compile error med en gang du er i Release og != .NET9 🥷 🚀